independent-publisher-connectors/Annature/README.md

```diff
@@ -0,0 +1,71 @@
+# Annature
+
+Annature is an Australia's leading digital signature provider, purpose built for all industries. It's an ISO 27001 certified technology company that builds digital signature, identity verification, and payment collection software.
+
+Consumers from small businesses to enterprises, use our software to manage their digital transactions online. This connectors supports only core operations like Getting Documents, Creating Envelope, Sending to recipients. 
```

Comment:
@iddycol all files should be free of grammatical errors and typos. in the description you have "...This connectors...", did you meant to say "...This connector..."

Reply:
We have removed the long sentences and updated with simple description. Please review now.